pkg/auditd/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine:edge AS mirror
```

Comment:
ah I missed this - good point.  Let's use `alpine:edge@sha256:b45de39913783fcc1ee31e15bc9a3df4226a0a51111e00ca74ea8531c099d4df`